recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+iMACOSX_SDK_VERSION:  # [osx and x86_64]
```

Comment:
iMACOSX_SDK_VERSION : the i at the beginning looks like a typo. But this doesn't look to be used either.

Reply:
Nice catch. Thanks!